projects/helper/coreAssets.json

```diff
@@ -106,15 +106,17 @@
     "aBNBb": "0xbb1aa6e59e5163d8722a122cd66eba614b59df0d"
   },
   "polygon": {
-    "WMATIC": "0xfd28c7cea3c50a060cb4c0059e453c6d4dd9829d",
+    "WMATIC": "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
```

Comment:
please revert this change, it is going to break other adapters

Reply:
Reverted WMATIC back to original address and added WMATIC_2.

Reply:
thanks